contracts/interfaces/IBond.sol

```diff
@@ -26,7 +26,7 @@ interface IBond {
     error NotEnoughCollateral();
 
     /**
-        @notice Emitted when Bond tokens are converted by a borrower.
+        @notice Emitted when bond shares are converted by a lender.
```

Comment:
I'd be ok changing more of the wording to include "bond shares" instead of Bonds where it makes sense. Is this the only place? I think there are places we call the "lender" bond holders as well. Should we stick with one or the other?